pkg/util/trace_info.go

```diff
@@ -141,7 +141,7 @@ func (t *TraceInfo) makeThriftBatch(TraceIDHigh int64, TraceIDLow int64) *thrift
 			OperationName: fmt.Sprintf("vulture-%d", t.generateRandomInt(0, 100)),
 			References:    nil,
 			Flags:         0,
-			StartTime:     unixMicro(t.timestamp),
+			StartTime:     time.Now().UnixMicro(),
```

Comment:
Does this impact vulture's consistency checks?

Reply:
This should be the exact same value:

https://github.com/grafana/tempo/pull/1483/files#diff-64607fbb74bbb31f1d923cc9c1b4916f90ba904eb09a21320dbf7f2cf2dc59fdL13-L16

Reply:
I say this and I just noticed that tons of tests related to these timestamps are failing. Will dig deeper.

Reply:
yup, i see what i did now. good catch!